### PR TITLE
new: Added my Warp theme: dmilith

### PR DESCRIPTION
**Pull request by dmilith**
# Pull Request Template

## Discord username (optional)
dmilith#1284

If you're in our Discord server please include your formatted like so: elvis#4747.

This is so we can give you the Contributor Discord role.
Note: It must be labelled like so: username#4747 (make sure to include the #XXXX).

## Name of theme
dmilith

Please include the name of the theme.

## How Has This Been Tested?

Have your run the python script?

### CODE REVIEW

**Comment by elviskahoro** [comment]
Hey dmilith,

Thanks for opening this PR and adding a new theme. You're the first PR with a custom background image included. We aren't able to include the image because of licensing and also because we're trying to keep the repo size as small as possible.

Could you remove the background image from the PR but add a comment in the yaml with a link to where people could download it?

Sorry that it took so long to get back to you!

File: standard/dmilith.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+accent: '#52ad70' # Accent color for UI elements
+details: darker # Whether the theme is lighter or darker.
+foreground: '#bbbbbb' # The foreground color.
+background: '#001000' # Terminal background color
+
+background_image:
```
Comment (elviskahoro) [inline standard/dmilith.yaml:6]: Could you comment this section out and also include a comment with a link to where someone could download the picture?

Similar to here: https://github.com/warpdotdev/themes/blob/main/warp_bundled/dark_city.yaml#L7

**Review by elviskahoro — CHANGES_REQUESTED** [review]
Hey dmilith,

Thanks for opening this PR and adding a new theme. You're the first PR with a custom background image included. We aren't able to include the image because of licensing and also because we're trying to keep the repo size as small as possible.

Could you remove the background image from the PR but add a comment in the yaml with a link to where people could download it?

Sorry that it took so long to get back to you!

File: standard/dmilith.yaml (hunk 1 above)
Comment (elviskahoro) [reply standard/dmilith.yaml:6]: To be clear, just the background image section not foreground and background haha.

**Comment by dannyneira** [comment]
@dmilith Any updates on the background image comment above? We'd really like to merge this and add it to the https://github.com/daltonmenezes/aura-theme repo for others to enjoy.

**Comment by dmilith** [comment]
@dannyneira  The image was taken from the free image source. If you want to save the colours, just drop the image background. (Support for the transparent terminal in Warp would be better than this)

**Comment by elviskahoro** [comment]
@dmilith can you remove the standard/dmilith_bg.jpg file and then we'll go ahead and merge it. Thanks again!

**Pull request closed without merging**